common.props

```diff
@@ -0,0 +1,38 @@
+<Project>
+  
+  <PropertyGroup>
+    <BuildVersion>3.4.0</BuildVersion>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+    <SignAssembly>true</SignAssembly> 
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Castle Project Contributors</Authors>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/castleproject/Windsor</RepositoryUrl>
+    <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0.html</PackageLicenseUrl>
+    <PackageProjectUrl>http://www.castleproject.org/projects/windsor/</PackageProjectUrl>
+    <PackageIconUrl>http://www.castleproject.org/img/windsor-logo.png</PackageIconUrl>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageOutputPath>../../Packages/</PackageOutputPath>
+    <PackageVersion>$(BuildVersion)</PackageVersion>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>false</IncludeSource>
+    <Copyright>Copyright 2004 Castle Project - http://www.castleproject.org/ </Copyright>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);DOTNET45</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\BreakingChanges.txt" />
```

Comment:
I need to specify the `ContentTargetFolders` property or a `PackagePath` for the content files (by default these files end up in both "content" and "contentFiles"). If I set the `ContentTargetFolders` to be the root of the package, there is an issue with the LICENSE file, because it doesn't have an extension. Trying to open the resulting package with NuGet package Explorer gives the error: "Part URI is not valid per rules defined in the Open Packaging Convention specification". 
So there are two options:
1.Put these files in a folder named "content" or "contentFiles" or whatever name
2.Rename LICENSE to LICENSE.txt so that these files can be put in the root folder.